requirements/plugins.txt

```diff
@@ -11,3 +11,4 @@ tutor-minio>=17.0.0,<18.0.0
 tutor-notes>=17.0.0,<18.0.0
 tutor-webui>=17.0.0,<18.0.0
 tutor-xqueue>=17.0.0,<18.0.0
+tutor-indigo>=17.0.0,<18.0.0
```

Comment:
Could you please move indigo such that the list of plugins is sorted?

Reply:
done